package.json

```diff
@@ -29,15 +29,15 @@
     "commitplease": "2.3.1",
     "core-js": "2.2.2",
     "cross-spawn": "2.2.3",
+    "eslint-config-jquery": "github:markelog/eslint-config-jquery",
```

Comment:
I'd prefer if you could do it here if possible. I know you might not plan to remove it but accidents happen and I think there's value in trying to make sure even old commits can be build without depending on data that may change or disappear. The npm registry is good in that regard because it doesn't allow to overwrite package versions and now it even made it much more difficult to unpublish packages.

If there's no time to publish it as `eslint-config-jquery` before landing I think even just publishing a `temporary-markelog-eslint-config-markelog` package and using that for a while would be safer. :) As the package wouldn't disappear and GitHub repos can.

Or just publish `eslint-config-jquery@0.0.1`, I don't think the jQuery team would be opposed to starting from that and just iterating later.